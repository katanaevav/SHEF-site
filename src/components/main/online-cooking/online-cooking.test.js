import React from "react";
import renderer from "react-test-renderer";
import OnlineCooking from "./online-cooking.jsx";

it(`Render Online Cooking`, () => {
  const tree = renderer.create(
      <OnlineCooking />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
