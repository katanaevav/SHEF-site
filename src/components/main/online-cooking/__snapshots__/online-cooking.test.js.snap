// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Online Cooking 1`] = `
<main
  className="page-main"
>
  <div
    className="page-main__wrapper"
  >
    <section
      className="order-head"
    >
      <div
        className="order-head__left-panel"
      >
        <h1
          className="order-head__header"
        >
          Онлайн кулинария
        </h1>
        <p
          className="order-head__text"
        >
          Хочется домашней еды или необходимо накрыть стол к празднику, а готовить нет времени или желания? Не беда - выбирайте блюда, а мы приготовим и все привезем!
        </p>
        <ul
          className="order-head__terms"
        >
          <li
            className="order-head__term"
          >
            Время заказа: 
            <span
              className="order-head__term-dd"
            >
              <br />
              До 17:00 текущего дня
            </span>
          </li>
          <li
            className="order-head__term"
          >
            Доставка: 
            <span
              className="order-head__term-dd"
            >
              <br />
              На следующий день
            </span>
          </li>
          <li
            className="order-head__term"
          >
            Минимальная сумма заказа: 
            <span
              className="order-head__term-dd"
            >
              <br />
              2900 рублей*
            </span>
          </li>
        </ul>
        <p
          className="order-head__text-term"
        >
          * Доставка в пределах МКАД 500р, 
          <br />
          за МКАД и до 25км от МКАД - 1000р.
        </p>
      </div>
      <div
        className="order-head__right-panel"
      >
        <img
          alt="Логотип"
          className="order-head__logo-image order-head__logo-image--online-cooking"
          src="./img/online-cooking.png"
          srcSet="./img/online-cooking@2x.png 2x"
        />
      </div>
    </section>
    <section
      className="order-navigator"
    >
      <h2
        className="order-navigator__header visually-hidden"
      >
        Навигация по меню
      </h2>
      <ul
        className="order-navigator__list"
      >
        <li
          className="order-navigator__item"
        >
          <a
            href="#1"
          >
            Первые блюда
          </a>
        </li>
        <li
          className="order-navigator__item"
        >
          <a
            href="#2"
          >
            Вторые блюда
          </a>
        </li>
        <li
          className="order-navigator__item"
        >
          <a
            href="#3"
          >
            Гарниры
          </a>
        </li>
        <li
          className="order-navigator__item"
        >
          <a
            href="#"
          >
            Десерты
          </a>
        </li>
        <li
          className="order-navigator__item"
        >
          <a
            href="#"
          >
            Безалкогольные напитки
          </a>
        </li>
        <li
          className="order-navigator__item"
        >
          <a
            href="#"
          >
            Вина
          </a>
        </li>
        <li
          className="order-navigator__item"
        >
          <a
            href="#"
          >
            Крепкий алкоголь
          </a>
        </li>
        <li
          className="order-navigator__item"
        >
          <a
            href="#"
          >
            Пицца
          </a>
        </li>
      </ul>
    </section>
    <section
      className="menu-group-navigator"
      id="1"
    >
      <h2
        className="menu-group-navigator__header"
      >
        Первые блюда
      </h2>
      <div
        className="menu-group-navigator__wrapper"
      >
        <ul
          className="menu-group-navigator__list"
        >
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                250 гр.
              </p>
              <p
                className="dish-tag"
              >
                NEW
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Анатомия вкуса
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                250 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                300 гр.
              </p>
              <p
                className="dish-tag"
              >
                HOT
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Суп Харчо с бараниной
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                350 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                350 гр.
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Борщ на говядине
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                290 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                350 гр.
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Солянка
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                210 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                320 гр.
              </p>
              <p
                className="dish-tag"
              >
                %
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Лапша домашняя
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                170 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
        </ul>
      </div>
    </section>
    <section
      className="menu-group-navigator"
      id="2"
    >
      <h2
        className="menu-group-navigator__header"
      >
        Вторые блюда
      </h2>
      <div
        className="menu-group-navigator__wrapper"
      >
        <ul
          className="menu-group-navigator__list"
        >
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                250 гр.
              </p>
              <p
                className="dish-tag"
              >
                NEW
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Анатомия вкуса
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                250 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                300 гр.
              </p>
              <p
                className="dish-tag"
              >
                HOT
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Суп Харчо с бараниной
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                350 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                350 гр.
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Борщ на говядине
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                290 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                350 гр.
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Солянка
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                210 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                320 гр.
              </p>
              <p
                className="dish-tag"
              >
                %
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Лапша домашняя
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                170 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
        </ul>
      </div>
    </section>
    <section
      className="menu-group-navigator"
      id="3"
    >
      <h2
        className="menu-group-navigator__header"
      >
        Гариниры
      </h2>
      <div
        className="menu-group-navigator__wrapper"
      >
        <ul
          className="menu-group-navigator__list"
        >
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                250 гр.
              </p>
              <p
                className="dish-tag"
              >
                NEW
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Анатомия вкуса
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                250 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                300 гр.
              </p>
              <p
                className="dish-tag"
              >
                HOT
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Суп Харчо с бараниной
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                350 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                350 гр.
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Борщ на говядине
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                290 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                350 гр.
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Солянка
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                210 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
          <li
            className="menu-group-navigator__item dish"
          >
            <div
              className="dish-wrapper dish-wrapper--head"
            >
              <p
                className="dish-weight"
              >
                320 гр.
              </p>
              <p
                className="dish-tag"
              >
                %
              </p>
            </div>
            <img
              alt="Анатомия вкуса"
              className="dish__image"
              src="./img/dish.png"
              srcSet="./img/dish@2x.png 2x"
            />
            <div
              className="dish__counter select-count"
            >
              <button
                className="select-count__button select-count__button--dec"
              >
                -
              </button>
              <input
                className="select-count__input"
                type="number"
                value="1"
              />
              <button
                className="select-count__button select-count__button--inc"
              >
                +
              </button>
            </div>
            <h3
              className="dish__header"
            >
              Лапша домашняя
            </h3>
            <p
              className="dish__description"
            >
              Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда Описание блюда
            </p>
            <div
              className="dish-wrapper dish-wrapper--footer"
            >
              <p
                className="dish__price"
              >
                170 р.
              </p>
              <button
                className="dish__to-cart"
              >
                В корзину
              </button>
            </div>
          </li>
        </ul>
      </div>
    </section>
  </div>
</main>
`;
